cmd/down.go

```diff
@@ -91,32 +92,50 @@ func runDown(ctx context.Context, dev *model.Dev) error {
 	spinner.Start()
 	defer spinner.Stop()
 
-	client, _, err := k8Client.GetLocalWithContext(dev.Context)
-	if err != nil {
-		return err
-	}
+	stop := make(chan os.Signal, 1)
+	signal.Notify(stop, os.Interrupt)
+	exit := make(chan error, 1)
 
-	if dev.Divert != nil {
-		if err := diverts.Delete(ctx, dev, client); err != nil {
-			return err
+	go func() {
+		client, _, err := k8Client.GetLocalWithContext(dev.Context)
+		if err != nil {
+			exit <- err
 		}
-	}
 
-	d, err := deployments.Get(ctx, dev, dev.Namespace, client)
-	if err != nil && !errors.IsNotFound(err) {
-		return err
-	}
+		if dev.Divert != nil {
+			if err := diverts.Delete(ctx, dev, client); err != nil {
+				exit <- err
+			}
+		}
 
-	trList, err := deployments.GetTranslations(ctx, dev, d, false, client)
-	if err != nil {
-		return err
-	}
+		d, err := deployments.Get(ctx, dev, dev.Namespace, client)
+		if err != nil && !errors.IsNotFound(err) {
+			exit <- err
+		}
 
-	err = down.Run(dev, d, trList, true, client)
-	if err != nil {
-		return err
-	}
+		trList, err := deployments.GetTranslations(ctx, dev, d, false, client)
+		if err != nil {
+			exit <- err
+		}
 
+		err = down.Run(dev, d, trList, true, client)
+		if err != nil {
+			exit <- err
+		}
+		exit <- nil
+	}()
+
+	select {
+	case <-stop:
+		log.Infof("CTRL+C received, starting shutdown sequence")
+		spinner.Stop()
```

Comment:
Do we need this call? or would the `defer` make the job?
https://github.com/okteto/okteto/pull/1641/files#diff-41918fbb88ea02aac469b573629ce8a0dc3010817fbb0b50e5e6509f22165f43R93
Same for other commands